klasses/admin.py

```diff
@@ -45,8 +45,20 @@ class BootcampRunEnrollmentAdmin(TimestampedModelAdmin):
 
     model = models.BootcampRunEnrollment
     include_created_on_in_list = True
-    list_display = ("id", "get_user_email", "bootcamp_run", "change_status", "active")
-    list_filter = ("change_status", "active", "bootcamp_run__bootcamp")
+    list_display = (
+        "id",
+        "get_user_email",
+        "bootcamp_run",
+        "change_status",
+        "active",
+        "user_certificate_is_blocked",
```

Comment:
kindly have a look at this comment: https://github.com/mitodl/bootcamp-ecommerce/issues/1135#issuecomment-762510159

Reply:
Yes, I changed to add a mark `user` here for better understanding.